app/views/includes/result-item.nunjucks

```diff
@@ -13,8 +13,8 @@
       {% if service.address.postcode %} {{ service.address.postcode }}<br />{% endif %}
       {% if service.contacts.telephoneNumber %}{{ service.contacts.telephoneNumber }}{% endif %}
       </p>
-    <p class="results__item__link"><a class="maplink" href="{{service.mapUrl}}">See map and directions</a></p>
-    <p class="results__item__link"><a class="serviceslink" href="{{ service.choicesServicesUrl }}">See this pharmacy's services</a></p>
+    <p class="results__item__link"><a class="maplink" href="{{service.mapUrl}}">Map and directions <span class="sr-only">for {{ service.name }} at {{ service.address.line1 }}</span></a></p>
```

Comment:
Is there a reason to have the space before the beginning of the span rather than within it?